homeassistant/components/proxmoxve/const.py

```diff
@@ -0,0 +1,62 @@
+"""Constants for ProxmoxVE."""
```

Comment:
I normally recommend that existing constants are moved in a preliminary PR.

Reply:
I created PR #90357 for this.